libs/api/domains/user-profile/src/lib/userProfile.service.ts

```diff
@@ -22,17 +32,71 @@ export class UserProfileService {
       .catch(handleError)
   }
 
-  async createUser(input: CreateUserProfileInput): Promise<UserProfile> {
-    const { ...createUserProfileDto } = input
+  async createUser(
+    input: CreateUserProfileInput,
+    nationalId: string,
+  ): Promise<UserProfile> {
+    const createUserDto: CreateUserProfileDto = {
+      nationalId: nationalId,
+      //temporary as schemas where not working properly
```

Comment:
still valid?

Reply:
yeah its still valid. But i lost my typing somehow. validated on the gql end.